python/foglamp/tasks/backup_restore/configuration_cache.json

```diff
@@ -0,0 +1 @@
+{"backup_dir": {"value": "/tmp", "description": "Directory where the backups will be created.", "type": "string", "default": "/tmp"}, "port": {"value": "5432", "description": "PostgreSQL port for backup and restore operations.", "type": "integer", "default": "5432"}, "database": {"value": "foglamp", "description": "Database to manage for backup and restore operations.", "type": "string", "default": "foglamp"}, "retention": {"value": "5", "description": "Number of backups to maintain, old ones will be deleted.", "type": "integer", "default": "5"}, "timeout": {"value": "1200", "description": "Timeout in seconds for the execution of the external commands.", "type": "integer", "default": "1200"}, "semaphores_dir": {"value": "/tmp", "description": "Directory used to store semaphores for backup/restore synchronization.", "type": "string", "default": "/tmp"}, "max_retry": {"value": "5", "description": "Number of retries for the operations.", "type": "integer", "default": "5"}, "host": {"value": "localhost", "description": "Host server for backup and restore operations.", "type": "string", "default": "localhost"}}
```

Comment:
We need to define the default location to store backups, @izoratti can you do this please?

Likewise for the semaphore storage.

Reply:
defined (with Ivan) details in : https://scaledb.atlassian.net/browse/FOGL-746